### PR TITLE
Awesome style list

### PR DESCRIPTION
**Pull request by godds**
Initial commit for https://github.com/finos/InnerSource/issues/51

### CODE REVIEW

File: docs/resources.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Tools
+
+- [FINOS InnerSource SIG Maturity Matrix](https://github.com/finos/InnerSource/tree/master/docs/maturity-matrix)
+- [SAP Project Portal for InnerSource](https://github.com/SAP/project-portal-for-innersource)
+
+## Training
+
+- [InnerSource Commons Learning Path](https://innersourcecommons.org/learn/learning-path/)
+  diff --git a/README.md b/README.md
```
Comment (amaltson) [inline docs/resources.md:38]: I think this is extra here...

**Review by amaltson — COMMENTED** [review]
Thank you so much for your contribution @godds!! This looks fantastic! Love the Awesome list, really well done and a fantastic start!

Just one super tiny extra line, otherwise this is ready to 🚀

File: docs/resources.md (hunk 1 above)
Comment (godds) [reply docs/resources.md]: Oops! Good spot. Should be fixed in commit below.

**Review by amaltson — APPROVED** [review]
LGTM

**Pull request merged**